onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1104,19 +1104,15 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                OpSchema::Optional)
         .Input(5,
                "seqlens_k",
-               // For prompt, the value is number of tokens (excluding padding) - 1.
-               "1d Tensor of shape (batch_size). Indicates past sequence lengths for token generation case.",
+               "1d Tensor of shape (batch_size). Indicates past sequence lengths for token generation case."
```

Comment:
This is not backward compatible. There are two options:
(1) keep the interface not change, and not use the input
(2) Change the interface, but need increase the version. Make sure our code could handle both v1 operator and v2 operator (just like we handle different version of onnx ops).